frontends/main/src/app-pages/ChannelPage/ChannelPage.tsx

```diff
@@ -11,21 +12,74 @@ import type {
   BooleanFacets,
 } from "@mitodl/course-search-utils"
 import { ChannelTypeEnum } from "api/v0"
-import LearningResourceDrawer from "@/page-components/LearningResourceDrawer/LearningResourceDrawer"
+import { useLearningResourceTopics } from "api/hooks/learningResources"
+import { ChipLink, Container, styled, Typography } from "ol-components"
+import { propsNotNil } from "ol-utilities"
+
+const SubTopicsContainer = styled(Container)(({ theme }) => ({
+  marginBottom: "60px",
+  [theme.breakpoints.down("sm")]: {
+    marginBottom: "24px",
+  },
+}))
+
+const SubTopicsHeader = styled(Typography)(({ theme }) => ({
+  marginBottom: "10px",
+  ...theme.typography.subtitle1,
+}))
+
+const ChipsContainer = styled.div({
+  display: "flex",
+  flexWrap: "wrap",
+  gap: "12px",
+})
 
 type RouteParams = {
   channelType: ChannelTypeEnum
   name: string
 }
 
+type SubTopicDisplayProps = {
+  parentTopicId: number
+}
+
+const SubTopicsDisplay: React.FC<SubTopicDisplayProps> = (props) => {
+  const { parentTopicId } = props
+  const topicsQuery = useLearningResourceTopics({
+    parent_topic_id: [parentTopicId],
+  })
+  const totalSubtopics = topicsQuery.data?.results.length ?? 0
+  const subTopics = topicsQuery.data?.results.filter(
+    propsNotNil(["channel_url"]),
+  )
+  return (
+    totalSubtopics > 0 && (
+      <SubTopicsContainer>
```

Comment:
The subtopic work Carey added in #1511 seems to have been accidentally removed in https://github.com/mitodl/mit-open/pull/1525. (But not the test, thankfully!). This adds it back.